tests/integration/test_charm.py

```diff
@@ -1,25 +1,33 @@
 import logging
-from pathlib import Path
-
-import yaml
 
 import pytest
+from runner import Runner
 
 log = logging.getLogger(__name__)
 
 
+async def file_contents(unit, path):
```

Comment:
Maybe useful to add this to pytest-operator.

Reply:
I agree, I dug this up from my previous charms. Do you think it belongs in pytest-operator or libjuju?